src/lib/modules/databases.js

```diff
@@ -29,11 +29,62 @@ DBManager.databaseOperation = function(operation, database, callback) {
     });
 };
 
+DBManager.buildForestsByHost = function(dbSettings) {
+    var count = dbSettings.forest['forests-per-host'];
+    // idenfity the hosts
+    var hosts = this.getConfiguration('hosts');
+    // build forest names
+    var forests = [], forestNames = [];
+    hosts.forEach(function(host, index) {
+        var hostSettings = common.objectSettings('hosts/' + host, this.env);
+        console.log('host is: ' + hostSettings['host-name']);
+        for (var i = 0; i < count; i++) {
+            forests.push({'host': hostSettings['host-name'], 'forest-name': dbSettings['database-name'] + '-' + (index * count + i)});
```

Comment:
While at this, you could apply the same naming convention that Roxy applies: dbname-hostnr-forestnr, where hostnr is zero-padded to 3 digits.. :)